setup.py

```diff
@@ -31,7 +31,6 @@
             'README.rst'
         )
     ).read(),
-    test_suite='nose.collector',
```

Comment:
This was for `setup.py test`, which is now deprecated.

Reply:
yes, no problem